src/components/DivElement.js

```diff
@@ -1,6 +1,16 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super(content);
+    this.content = content;
```

Comment:
You should be taking advantage of the parent constructor to handle variable assignment. HTMLElement is expecting a tag and then the content, so by passing in only the content var you are assigning content to both variables. 
super('div', content); is what you are looking for.